Test/client_test.py

```diff
@@ -57,6 +57,13 @@ def test_edit_org(self):
         response = self.c.edit_org(orgname,obj)
         self.assertEqual(204,response.status_code)
 
+    def test_get_supported_languages(self):
+        client = Algorithmia.client(api_key=os.environ.get('ALGORITHMIA_API_KEY'))
+        response = client.get_supported_languages()
+        if("error" in response):
+            print(response)
+        self.assertTrue(response is not None and "name" in response[0])
```

Comment:
Is it possible for us to test for a specific language here? We could certainly use the fastAPI testing harness that we integrated for exception handling